packages/amplify_authenticator/pubspec.yaml

```diff
@@ -10,9 +10,9 @@ environment:
   flutter: ">=2.10.0"
 
 dependencies:
-  amplify_auth_cognito: ^0.5.0
-  amplify_core: ^0.5.0
-  amplify_flutter: ^0.5.0
+  amplify_auth_cognito: ^0.6.4-rc.1
```

Comment:
nit: because authenticator will not be released here, is this needed? Doesn't seem to do any harm.

Reply:
Good catch. Yeah, we shouldn't need to change the Authenticator constraints at all

Reply:
I guess makes sense to have this so everything matches.